ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -187,10 +187,13 @@ class ActionBar extends React.Component {
         staticActionVariables: results.staticActionVariables,
       });
 
-      this.thNotify.send(
-        `Request sent to create an interactive job via actions.json.
-          You will soon receive an email containing a link to interact with the task.`,
-        'success');
+      this.thNotify.send('Request sent to create an interactive job via actions.json.', 'success');
+
+      Object.assign(window.open(), {
+        opener: null,
+        location: getInteractiveTaskUrl(taskId),
+        target: '_blank',
```

Comment:
Could you set `noopener` here?
https://developer.mozilla.org/en-US/docs/Web/API/Window/open#noopener
https://mathiasbynens.github.io/rel-noopener/

Reply:
`opener: null` takes care of `noopener`. See the `opener` section under https://developer.mozilla.org/en-US/docs/Web/API/Window/open#Window_functionality_features.